packages/format-currency/src/index.ts

```diff
@@ -28,8 +31,17 @@ export default function formatCurrency(
 	if ( ! currencyDefaults || isNaN( number ) ) {
 		return null;
 	}
-	const { decimal, grouping, precision, symbol } = { ...currencyDefaults, ...options };
+	const { decimal, grouping, precision, symbol, isSmallestUnit } = {
```

Comment:
I wonder if we could figure out a better name than `isSmallestUnit`... Is there a generic term for the coin of a currency that we could use here? `coin` is one, but I guess it also adds some ambiguity?

Also, are there currencies with more than one smaller unit than the primary one? I'd trust y'all payments folks would be aware if there's one but just wanted to make sure it's been considered.

Reply:
Stripe calls it [smallest currency unit](https://stripe.com/docs/currencies#zero-decimal), which is where I got the term. Probably because of that we call it the "smallest unit" on our backend, although in the shopping cart we just call it "integer" (eg: `original_price_integer`). I did consider calling this `isInteger` but I felt that was probably even more ambiguous than `isSmallestUnit`. 🤷

Reply:
Thanks for confirming 👍 Naming is hard 😅